src/ApparentHorizons/ComputeHorizonVolumeQuantities.hpp

```diff
@@ -68,12 +68,6 @@ struct ComputeHorizonVolumeQuantities
       const InverseJacobian<DataVector, 3, Frame::ElementLogical, TargetFrame>&
           inverse_jacobian);
 
-  using allowed_src_tags = tmpl::list<
```

Comment:
Why is CCE using ComputeHorizonVolumeQuantities?   CCE should use a different class that conforms to `intrp::protocols::ComputeVarsToInterpolate`.   (Similarly, interpolation onto an excision bdry uses a different class as well).  Presumably whatever CCE needs should be simpler than ComputeHorizonVolumeQuantities because CCE (I think) needs only metric and 1st derivs, not 2nd derivs.

If it so happens that CCE needs (a subset of) `ComputeHorizonVolumeQuantities` as-is, then I can see the case for not writing another class with a subset of the functionality.  But in that case, `ComputeHorizonVolumeQuantities` should be renamed.  (and I am fine with removing `allowed_src_tags` to allow passing in a Variables with extra tags).

(Also, note that by removing `allowed_src_tags`, the user might assume that any Tags in the SrcVariables that are not in `required_src_tags` remain unused.  But that is not true. I don't know how to get around this other than better dox.)